models.yaml

```diff
@@ -326,6 +326,28 @@ models:
       url: !secret ollama_url
       model: llama3:latest
 
+  - model_id: llama3.3
+    domain: ollama
+    description: Llama 3.3 (70B) from Meta using Ollama
+    urls:
+      - https://ollama.com/library/llama3
+    config_entry_data:
+      url: !secret ollama_url
+      model: llama3.3:latest
+    config_entry_options:
+      llm_hass_api: assist
+
+  - model_id: llama3.3-awq
+    domain: vicuna_conversation
+    description: Llama 3.3 AWQ running in an OpenAI API-compatible server
```

Comment:
Can you add some urls with references to this AWQ model and/or explain what this means?

Reply:
added some URLs, AWQ has a minor reduction in quality according to the evals but also is about 6GB smaller in weights.

Reply:
Thanks!